### ISSUE TITLE
JsonNode

### ISSUE THREAD

**Issue by thepian** [vojtechhabarta/typescript-generator#289]
What would be the best way to handle com.fasterxml.jackson.databind.JsonNode ?

```
class LogData {

    private ArrayList<JsonNode> data;
}
```

Can I add an annotation to make it persist as a plain Object?

Or could the generator be improved?

**Comment by vojtechhabarta** [comment]
I was thinking that `JsonNode` is already supported. It should be treated as `Object` so it should be mapped to `any` type in TypeScript. But actually this only works for `JsonNode` used directly but not in `List` or other "containers".

For this class:
``` java
class ClassWithJsonNode {
    public JsonNode node;
    public List<JsonNode> nodes;
}
```
following output is currently generated:
``` typescript
interface ClassWithJsonNode {
    node: any;
    nodes: JsonNode[];  // should be any[]
}
interface JsonNode extends Base, TreeNode, Iterable<JsonNode> {
}
// etc.
```

I will try to fix this.

You can workaround this by excluding `com.fasterxml.jackson.databind.JsonNode` class. (All excluded classes are mapped to `any`.)

**Comment by vojtechhabarta** [comment]
Released [v2.9.456](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.9.456).
